### ISSUE TITLE
rclone mount path does not exist

### ISSUE THREAD

**Issue by Panagiotis-Zachos** [rogerfar/rdt-client#519]
**Version 2.0.79.0**

**OS: Windows 11**

**Docker**

**Real-debrid**

**Symlink Downloader**

I can't find the log file, but the error I'm getting is:

`Unable to start download: An unexpected error occurred preparing download https://22.download.real-debrid.com/d/CLRVV3JXGRP64/Franklin%202024%20S01E04%20Small%20Revenge%201080p%20ATVP%20WEB-DL%20DDP5%201%20Atmos%20H%20264-FLUX%5BEZTVx.to%5D.mkv for torrent Franklin 2024 S01E04 Small Revenge 1080p ATVP WEB-DL DDP5 1 Atmos H 264-FLUX[EZTVx.to].mkv: Mount path z:\torrents does not exist! for download Franklin 2024 S01E04 Small Revenge 1080p ATVP WEB-DL DDP5 1 Atmos H 264-FLUX[EZTVx.to].mkv. Completed: 0%, avg speed: 0bytes/s (4604400a-25a8-4787-bfac-3655a29da0e5) remoteID:  for torrent Franklin 2024 S01E04 Small Revenge 1080p ATVP WEB-DL DDP5 1 Atmos H 264-FLUX[EZTVx.to].mkv (YUWYH2UGNWDZQ - downloaded 100%) (1ea7389c-d500-420d-b6a4-03869764e894)`

To run the container I'm using the following command:

`docker run --pull=always --volume e:/Torrents/:/data/downloads --volume e:/Torrents/:/data/db --log-driver json-file --log-opt max-size=10m -p 6500:6500 --name rdtclient rogerfar/rdtclient:latest`

rclone is mounted at drive z: using the following command:

`mount zurg: Z: --dir-cache-time 30s`

I can access the files normally and copy them or play them without any problems. I'm not very familiar with docker, however I tried mounting Z:/torrents/ as a volume in the docker run command, and was getting errors that it is not a supported filesystem, I tried mounting rclone as a network drive but that didn't work either. 

Do you have any suggestions?

**Comment by khazardstudent** [comment]
use --allow-other flag with rclone

**Comment by AnCasper** [comment]
Having the same issue:
![Screenshot 2024-08-09 195522](https://github.com/user-attachments/assets/d6479efe-f92d-4c2f-9ed9-f6cd99912b6e)

![Screenshot 2024-08-09 195619](https://github.com/user-attachments/assets/19b739d8-939f-4c1a-b202-b2d1d7f00882)

Does exist and is accesable: RDT running as Admin, just to be sure
![Screenshot 2024-08-09 195959](https://github.com/user-attachments/assets/9b979923-08b8-46b9-9946-442da2898541)

Should also note that this is also an issue when it comes to Symlinking the files. RDT trys to sym links that not in root of mount instead of the __all__ folder.

**Comment by Bugsbugme** [comment]
I am having the same problem but with the windows exe. I'm using zurg + rclone to mount.
I had it working for a while but now it just gives me this error. It was working at first with just rclone but the performance was bad so I added zurg. I've tried going back to just rclone but that didn't work either.
I've tried changing the permissions for the drive, the settings don't seem to stick though, if that's even the problem. Also tried running as admin.

My log file:
[rdtclient.log](https://github.com/user-attachments/files/16838181/rdtclient.log)

Another weird thing it's doing is it's creating an empty folder for each torrent inside the folder with the executable.
